package.json

```diff
@@ -7,6 +7,7 @@
   },
   "scripts": {
     "start": "backstage-cli repo start",
```

Comment:
does _this_ command break when there are several `"role": "frontend"`? Or is that ok